### PR TITLE
HW 10.1 Solution added

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 1]
```diff
@@ -17,7 +25,20 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        List<String> list = new ArrayList<>();
+        try {
+            String unsorted = Files.readString(Paths.get(fileName));
+            String[] letters = unsorted.toLowerCase().replaceAll("[^a-z]+", "").split("");
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: let's use array of characters

Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: Fixed.

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 2]
```diff
@@ -17,7 +24,18 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        List<Character> list = new ArrayList<>();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: do we need this list? let's remove it

Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: Fixed

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**